sdk/go/common/workspace/project.go

```diff
@@ -116,23 +116,16 @@ func (proj *Project) Save(path string) error {
 	contract.Require(path != "", "path")
 	contract.Require(proj != nil, "proj")
 	contract.Requiref(proj.Validate() == nil, "proj", "Validate()")
-
-	m, err := marshallerForPath(path)
-	if err != nil {
-		return err
-	}
-
-	b, err := m.Marshal(proj)
-	if err != nil {
-		return err
-	}
-
-	return ioutil.WriteFile(path, b, 0644)
+	return save(path, proj, false /*mkDirAll*/)
 }
 
 type PolicyPackProject struct {
 	// Runtime is a required runtime that executes code.
 	Runtime ProjectRuntimeInfo `json:"runtime" yaml:"runtime"`
+	// Version specifies the version of the policy pack. If set, it will override the
+	// version specified in `package.json` for Node.js policy packs.
+	Version string `json:"version,omitempty" yaml:"version,omitempty"`
```

Comment:
I didn't quite follow why this needs to be added as part of these changes?

Reply:
We need a place to specify the version for Python and `PulumiPolicy.yaml` seemed like a logical place to do that. For Node.js, you can continue to specify the version in `package.json`, but if it is specified in `PulumiPolicy.yaml` the `PulumiPolicy.yaml` value will be used.